src/views/Simulation/utils/MinimapContainer.js

```diff
@@ -126,22 +140,24 @@ export class MinimapContainer extends Container {
     this.dragTarget = null;
   }
 
-  cloneContainer(containerSource) {
-    const containerClone = new Container();
+  forgeContainer(containerSource) {
+    const containerForged = new Container();
+    const ratio = this.getSceneRatio();
 
     containerSource.children.forEach((child) => {
-      if (child instanceof Graphics) {
-        const childClone = child.clone(true);
-        childClone.position.set(child.x, child.y);
-        containerClone.addChild(childClone);
-      } else if (child instanceof Container) {
-        const childClone = this.cloneContainer(child);
-        childClone.position.set(child.x, child.y);
-        containerClone.addChild(childClone);
+      if (Object.values(NODE_LABELS).includes(child.label)) {
+        const nodeSprite = new Sprite(this.textures[child.label]);
+        nodeSprite.position.set(child.x * ratio, child.y * ratio);
+        containerForged.addChild(nodeSprite);
+        containerForged.position.set(child.x * ratio, child.y * ratio);
+      } else {
+        const childContainer = this.forgeContainer(child);
+        childContainer.position.set(child.x * ratio, child.y * ratio);
+        containerForged.addChild(childContainer);
```

Comment:
Do we want to have this `else` block? If I understand correctly, removing this block would cause to the labels and links not being drawn in the minimap. Isn't it what we want?

Reply:
If the child is not a node type, we must do a recurs on this child to search other node types.

Reply:
Oh, ok it makes sense if it is for container, thank you :)
In this case, do we really need to apply the scene ratio for children elements inside the container? (because the parent container is already positioned)

Reply:
I confirm it don't work without _ratio_ factor.